core/governance/market.go

```diff
@@ -232,20 +232,19 @@ func buildMarketFromProposal(
 		tsTimeWindow, _ := netp.GetDuration(netparams.MarketTargetStakeTimeWindow)
 		tsScalingFactor, _ := netp.GetDecimal(netparams.MarketTargetStakeScalingFactor)
 		// get triggering ratio
-		triggeringRatio, _ := netp.GetDecimal(netparams.MarketLiquidityTargetStakeTriggeringRatio)
-
-		params := &types.TargetStakeParameters{
+		lmParams := &proto.LiquidityMonitoringParameters{}
```

Comment:
This seems redundant, instantiating an empty struct will result in default values at best or errors when trying to parse decimals form empty strings (not sure what the behaviour is).

I'd remove this entire section, these aren't marked as default parameters, so I believe the proto validation will assure these are populated.